packages/client/src/plugins/TestingPlugin2.ts

```diff
@@ -0,0 +1,35 @@
+import type { SyftEvent, ISyftPlugin, IReflector } from '../types';
+import {MetricsDependency} from "../metrics_dependency_registry";
+
+export class TestingPlugin2 implements ISyftPlugin {
```

Comment:
why another testing plugin ?

Reply:
i can remove this and just use a mock/spy in the tests. just want to make sure that multiple plugins show up in the registry